WordPress/gutenberg
# FocalPointPicker: Apply modern styling

## What?


## Testing Instructions

1. Open a post or page. 
2. Insert a media and text block.
3. Add an image. 
4. Open block inspector.
5. See changes. 

## Screenshots or screencast 

### Current: 

https://github.com/WordPress/gutenberg/assets/1813435/cf268f24-6ae1-4b5e-b7f0-5325f75bffd8

### Proposed: 

https://github.com/WordPress/gutenberg/assets/1813435/b43d4b61-0332-45bd-a976-94db2f924410
